includes/admin_menu.php

```diff
@@ -39,3 +39,16 @@ function tsml_fix_highlight($parent_file)
     }
     add_filter('parent_file', 'tsml_fix_highlight');
 });
+
+// Add a Widget to the main WordPress Dashboard Page
+add_action('wp_dashboard_setup', 'tsml_dashboard_widgets');
+function tsml_dashboard_widgets()
+{
+    global $wp_meta_boxes;
+
+    wp_add_dashboard_widget('tsml_help_widget', 'Code for Recovery/12 Step Meeting List(TSML) Plugin', 'tsml_dashboard_help', null, null, 'normal', 'high');
+}
+function tsml_dashboard_help()
+{
+    echo '<p><a href="https://code4recovery.org/">Code for Recovery</a> ' . __('is a nonprofit organization of volunteer members building technology services for recovery fellowships, such as AA and Al-Anon. If you need help, please join our discussion forum. If you would like to make a tax-deductible contribution, please', '12-step-meeting-list') . ' <a href="https://code4recovery.org/"> ' . __('visit our website', '12-step-meeting-list') . '.</a></p>';
```

Comment:
this isn't a great construction for translations, because it means that "visit our website" has to come at the end of the string, right before the period. as it says in the [documentation](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#best-practices-for-writing-strings), it's better to translate the whole sentence because word order can vary from language to language

Reply:
Reverted to current develop branch which doesn't have this code.

Reply:
This is another side effect, because this is code that I had in #596. I did some more reading, and I think I see a better way to handle translation for this paragraph. The html code from the hyperlinks threw me off. I'll push a new commit, hopefully in the next day or two, into that other PR, to hopefully do that translation string better